tools/hpo/sweep.py

```diff
@@ -22,14 +22,17 @@
 from omegaconf import DictConfig, ListConfig, OmegaConf
 from pytorch_lightning import seed_everything
 from pytorch_lightning.loggers import WandbLogger
+from utils import flatten_hpo_params
```

Comment:
I previously got a linting error because of this. Does this work?

Reply:
I added `__init__.py` to `tools/benchmarking` so it works now. The issue is that `astroid` (pylint's base) cannot distinguish two folders with the same name in the same hierarchy (tools). But apparently it works when` __init__.py` is provided. I am not sure why would the two folders being a package make a difference.